dbus/dbus_libgio.go

```diff
@@ -55,6 +55,12 @@ func (d *dbusAPILibGio) BusGet(busType uint) (Handle, error) {
 	if Handle(gerror) != nil {
 		return Handle(nil), ErrorFromNative(Handle(gerror))
 	}
+
+	// For most applications it makes sense to close when the connection to
+	// the session ends. But Mender should keep running so that a broken
+	// DBus setup does not prevent the device from being updated.
+	C.g_dbus_connection_set_exit_on_close(conn, C.gboolean(0))
```

Comment:
Oh, good catch, this is the one which I could not understand: the Mender client quit when I stopped the dbus service!

Reply:
Exactly what happened to me as well! :-)